requirements.txt

```diff
@@ -0,0 +1,4 @@
+beautifulsoup4==4.12.0
+requests==2.28.2
+selenium==4.8.3
```

Comment:
try not rework to static website scrapping, although it is great you tried selenium